### ISSUE TITLE
Bromite does not keep webpages open in the background and the pages reload upon switch (full windows as well as split mode)

### ISSUE THREAD

**Issue by vdbhb59** [bromite/bromite#2544]
### Preliminary checklist

- [X] I have read the [README](https://github.com/bromite/bromite/blob/master/README.md).
- [X] I have searched the existing issues for my problem. This is a new ticket, NOT a duplicate or related to another open issue.
- [X] I have read the [FAQs](https://github.com/bromite/bromite/blob/master/FAQ.md).
- [X] I have updated Bromite to the latest version. The bug is reproducible on this latest version.
- [X] This is a bug report about the Bromite browser (not about the website, building Bromite, F-Droid or anything else).

### Can the bug be reproduced with corresponding Chromium version?

No

### Bromite version

108.0.5359.156

### Device architecture

arm64-v8a

### Android version

13

### Device model

OnePlus 5, Vivo Y35 (V2205), LG Wing 5 (Android 12)

### Changed flags

no flags changed (another issue here, opening chrome://flags in Bromite, first freezes the page, and after 8-10 seconds, crashes bromite with runtime exception. This is not reproduced in Chromium. I am using the same version for both the browsers.

### Is this bug about the SystemWebView?

No

### Is this bug happening in an incognito tab?

Yes

### Is this bug caused by the adblocker?

No

### Is this bug a crash?

No

### Describe the bug

Reproducing steps as well as bug description.
1. Launch Bromite
2. Open any webpage, like Yahoo, Amazon, Foosbyte, Blogs, anything
3. Run the page for let's say whatever time you wish to, and then switch to another application.
4. Change to any other application which will open in front (be it split window or full window)
5. Stay on this application for let's say 4-5 maybe even more minutes, but minimum of 3-5 minutes
6. Now switch to the bromite application, which had your last page running
7. The page reloads, and bromite ideally is not able to keep the webpage in background

Notes:
a. Battery optimisation, background power, background running, smart battery control, etc, all of these are not blocking, or are turned off for Bromite, and this still happens.
b. RAM is not an issue, and this happens despite the fact that RAM is 16GB (so not about memory as well)
c. This is happening for quiet sometime now, but apparently it was not reported.

Not sure why this is happening, and this is causing every page to reload, which is not just annoying, but at times it causes data loss (like opening emails over browser, and ther is a need to switch between apps for more than a minute or so, that page to gets reloaded).

### Steps to reproduce the bug

1. Launch Bromite
2. Open any webpage, like Yahoo, Amazon, Fossbyte, Blogs, anything at all
3. Run the page for let's say whatever time you wish to, and then switch to another application.
4. Change to any other application which will open in front (be it split window or full window)
5. Stay on this application for let's say 4-5 maybe even more minutes, but minimum of 3-5 minutes
6. Now switch to the bromite application, which had your last page running (this can be even other pages/tabs, and they too get reloaded)
7. The page reloads (as per number 6 above, be any page on bromite, reloads), and bromite ideally is not able to keep the webpage in background

### Expected behavior

Pages should remain active in the background, and should not get reloaded. This used to work around 4-5 builds back, at least. I honestly do not recollect the last build it used to work on, but it started behaving like this since around July-August 2022.

**Comment by vdbhb59** [comment]
Below is a comment from a user named "gaiden" from Matrix F-Droid Chat, and requested for me to add it to this bug report. If based on this comment/details from gaiden, I need to open a new bug report, kindly lett me know the same.

> On the newest version of Bromite 108.0.5359.156
When I choose Restrict Background Activity on Battery Saver option (Android's App Info), Bromite will be closed automatically after just around 1 to 2 minutes on the background.
This never happened before (6 months to 12 months ago).
When I choose Battery Saver (Recommended) = MIUI Battery Saver, Bromite will stay on the background but loose the internet connection every 1 or 2 minutes on the background. Then it will reconnected to Internet in 6 seconds when we reopenned it.
I have tried this a few times in a row and Bromite will stay on the background, just need to reconnected.
So the dev have changed the behaviour when we use Restrict Background Activity.
Android 11 / MIUI 12.5.5 on Redmi Note 9
